app/src/main/java/com/example/snookerapi/adapters/PlayerAdapter.kt

```diff
@@ -6,75 +6,45 @@ import android.view.ViewGroup
 import androidx.recyclerview.widget.DiffUtil
 import androidx.recyclerview.widget.ListAdapter
 import androidx.recyclerview.widget.RecyclerView
-import com.example.snookerapi.model.PagingData
-import com.example.snookerapi.model.Player
-import com.example.snookerapi.databinding.ItemLoadingBinding
 import com.example.snookerapi.databinding.ItemPlayerBinding
+import com.example.snookerapi.model.Player
 
 class PlayerAdapter(
     context: Context,
     private val onPlayerClicked: (Player) -> Unit,
-) : ListAdapter<PagingData<Player>, RecyclerView.ViewHolder>(DIFF_UTIL) {
+) : ListAdapter<Player, RecyclerView.ViewHolder>(DIFF_UTIL) {
```

Comment:
для одного типа элемента у тебя 1 вью холдер, можно сделать так и потом не нужно будет холдер кастовать 
```suggestion
) : ListAdapter<Player, PlayerViewHolder>(DIFF_UTIL) {
```

Reply:
Done